Interpolater.h

```diff
@@ -0,0 +1,132 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2021, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Description: Computes the Interpolater algorithm on the given snapshot matrix.
```

Comment:
Interpolater or interpolator? According to the dictionary, both are acceptable English words, but it seems interpolator is commonly used in mathematics. A google search for interpolator shows many results, but a google search for interpolater asks "Did you mean interpolator?" There are many instances to change, but if you agree that interpolator is the more common choice then it would be better to make the change now rather than later.

Reply:
Oh, I didn't know there were two different spellings lol. I don't have a preference, so I'll switch to interpolator.